packages/connect-examples/new-example/app/components/documentation/ArchitectureDiagram.tsx

```
@@ -0,0 +1,265 @@
+          aria-label="Maximize diagram"
+        >
+          <Maximize className="h-4 w-4" />
+        </Button>
+      </div>
+
+      {isModalOpen && (
+        <div
+          className="fixed inset-0 z-50 bg-black/80 flex items-center justify-center p-4"
+          onClick={() => setIsModalOpen(false)}
+          onKeyDown={e => e.key === 'Escape' && setIsModalOpen(false)}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)